Source/Applications/SystemCenterNotification/Scripts/TSX/EmailCategory/EmailCategoryPage.tsx

```diff
@@ -50,40 +42,73 @@ const EmailCategoryPage = (props: IProps) => {
     const category = useAppSelector((state) => EmailCategorySlice.Datum(state, parseInt(props.useParams.id)));
     const status: Application.Types.Status = useAppSelector(EmailCategorySlice.Status);
 
-    const [tab, setTab] = React.useState<tab>('settings');
+    const [tab, setTab] = React.useState<Tab>(getTab());
+    const Tabs = [
+        { Label: 'General Settings', Id: 'settings' },
+        { Label: ' Emails', Id: 'emails' },
+    ];
```

Comment:
const can be moved external, since it doesn't mutate and is the same every render